newIDE/app/src/EventsFunctionsExtensionEditor/EventsFunctionConfigurationEditor/ValueTypeEditor.js

```diff
@@ -146,34 +153,10 @@ export default function ValueTypeEditor({
                     label={t`Created objects`}
                   />
                 )}
-                {!isExpressionType && (
-                  <SelectOption
-                    value="imageResource"
-                    label={t`Image resource (JavaScript only)`}
-                  />
-                )}
-                {!isExpressionType && (
-                  <SelectOption
-                    value="audioResource"
-                    label={t`Audio resource (JavaScript only)`}
-                  />
-                )}
                 {!isExpressionType && (
                   <SelectOption
                     value="jsonResource"
-                    label={t`JSON resource (JavaScript only)`}
-                  />
-                )}
-                {!isExpressionType && (
-                  <SelectOption
-                    value="fontResource"
-                    label={t`Font resource (JavaScript only)`}
-                  />
-                )}
-                {!isExpressionType && (
-                  <SelectOption
-                    value="bitmapFontResource"
-                    label={t`Bitmap font resource (JavaScript only)`}
+                    label={t`Resource (JavaScript only)`}
```

Comment:
Why keeping this one?

Reply:
Actually, instead of having the list of resource types in the main list, it uses an accessory drop-down list for the resource types, `jsonResource` being the default value.
As the resource type is directly inside the parameter type, I added some logic to split them (it's not pretty but it works).